processor/Cargo.toml

```diff
@@ -23,12 +23,12 @@ std = ["vm-core/std", "winter-prover/std", "log/std"]
 
 [dependencies]
 log = "0.4.14"
+assembly = { package = "miden-assembly", path = "../assembly", version = "0.5", default-features = false }
```

Comment:
As mentioned in the previous comment - I'd like to avoid making `processor` depend on `assembly` crate.

Reply:
Fixed!